framework/PageForm/PageForm.tsx

```diff
@@ -120,6 +120,22 @@ export function PageForm<T extends object>(props: {
           gap: 0,
         }}
       >
+        {error && (
+          <Alert
+            variant="danger"
+            title={error.message ?? ''}
+            isInline
+            style={{
+              paddingLeft: isMd && props.onCancel ? 24 : undefined,
+              position: 'sticky',
```

Comment:
i'd like to avoid sticky.
I think the alert should be outside the scrollable area of the form